todo_app/app.py

```diff
@@ -1,32 +1,44 @@
-from flask import Flask
-from flask import render_template, request, redirect, url_for
+from flask import Flask, render_template, redirect, url_for, request
 
+from todo_app.data import trello_items as trello
 from todo_app.flask_config import Config
-from todo_app.data.session_items import get_items
-from todo_app.data.session_items import add_item
-
 
 app = Flask(__name__)
-app.config.from_object(Config())
-
+app.config.from_object(Config)
 
 
 @app.route('/')
 def index():
-    # show all to-do items
-    items = get_items()
-    return render_template("index.html", items=items)
+    items = trello.get_items()
+    return render_template('index.html', items = items)
+
+
+@app.route('/items/new', methods=['POST'])
+def add_items():
+    name = request.form['name']
+    trello.add_item(name)
+    return redirect(url_for('index'))
+
 
+@app.route('items/<id>/start')
```

Comment:
A few of these urls throw an error for me that reminds me they need to start with a slash:
```suggestion
@app.route('/items/<id>/start')
```